week-2/Homework/mandatory/3-playing-computer.js

```diff
@@ -7,31 +13,44 @@
   Answer the following questions:
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
+
+      //because b is not defined with a value as x and a ....
+
   2. Remove the line that throws the error.
   3. What is printed to the console?
+            2     6    4    9   6    13   8
+      /// 2 the value of x , 6 the value of a , moving inside 'for loop' where i starts with value 0 and end with 4,
+     console d must be an even number and console e must be an odd number
+     first console d prints  4 where i=0 + x=2 + x=2 based on the f2 called function, 
+    then console e = 9 where i=1 + a=6 + the increment of a=2,,,,, back to the loop 6 is printed as value
+    of d where i=2 + x=2 + x=2,, console 13 is e when i=3 + a=6 + the increment a=4,,,,, last printed console is 8 
+      when i=4 + x=2 + x=2.
+
```

Comment:
you have been ask to tell what the out of each number belongs to. for example x = 2. Could fix this please

Reply:
done